appveyor.yml

```diff
@@ -46,12 +46,33 @@ install:
   - if [%BUILD_TYPE%]==[release] (set CARGO_RELEASE_ARG=--release)
 
 build_script:
+  - ps: echo $env:CHANNEL
+  - ps: |
+      If ($env:CHANNEL -eq "nightly" -And $env:APPVEYOR_REPO_TAG -eq "false") {
+         $env:CXX = "clang-cl.exe"
+         $env:CXXFLAGS = "--coverage"
+         $env:CARGO_INCREMENTAL = "0"
+         $env:RUSTFLAGS = "-Zprofile -Ccodegen-units=1 -Ctarget-feature=+crt-static"
+      }
   - cargo build %CARGO_RELEASE_ARG% --features=%FEATURES% --locked --target %TARGET%
-  - 7z a grcov-win-%PLATFORM_TARGET%.tar .\target\%TARGET%\%BUILD_TYPE%\grcov.exe
```

Comment:
Nit: You can keep this, even if it doesn't do anything, at least it makes us notice if we break something.